### DISCUSSION TITLE
Support for Prometheus remote write

### DISCUSSION THREAD

**Discussion by Sparc0** [karimra/gnmic#607] [Ideas]
Are there any plans for gnmic to support Prometheus remote write?
So i could use gnmic to read metrics from a Kafka topic and write to our Cortex db.

**Comment by karimra** [comment]
If I understand correctly, you want gnmic to write directly to cortex (using prometheus remote write functionality) ?
There is no plan for this at the moment, but I could add it if you are interested.

**Reply by Sparc0** [reply]
yes i would be interested in that feature.

**Reply by karimra** [reply]
@Sparc0 v0.26.0 supports prometheus remote write: https://gnmic.kmrd.dev/user_guide/outputs/prometheus_write_output/
let me know if you face some issues using it.

**Reply by Sparc0** [reply]
Thanks, will try it out right away.

**Reply by Sparc0** [reply]
Finally got around to try it and thanks again. 
It works as i expect :) 
👍
